packages/grid/demo/~patterns/stories/CardStory.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useContext } from 'react';
+import useResizeObserver from 'use-resize-observer';
+import { Story } from '@storybook/react';
+import { ThemeContext } from 'styled-components';
+import { PaneProvider, Pane } from '@zendeskgarden/react-grid';
+import { MD, LG } from '@zendeskgarden/react-typography';
+import { IColumns } from './types';
+
+const Row = ({ panes }: { panes: IColumns['panes'] }) => {
+  const themeContext = useContext(ThemeContext);
+  const { ref, width = 1, height = 1 } = useResizeObserver<HTMLDivElement>();
+
+  return (
+    <PaneProvider
+      totalPanesWidth={width}
+      totalPanesHeight={height - 20}
+      defaultColumnValues={{}}
+      defaultRowValues={{
+        'row-1': 1,
+        'row-2': 1,
+        'row-3': 1
+      }}
+    >
+      {({ getGridTemplateColumns, getGridTemplateRows }) => {
+        return (
+          <div
+            ref={ref}
+            style={{
+              direction: themeContext.rtl ? 'rtl' : 'ltr',
+              width: '100%',
+              height: '100%',
+              display: 'grid',
+              gap: '10px',
+              gridTemplateRows: getGridTemplateRows(),
+              gridTemplateColumns: getGridTemplateColumns()
+            }}
+          >
+            {panes.map(pane => (
+              <Pane key={pane.name}>
+                <Pane.Content>
+                  <div style={{ padding: '8px' }}>
+                    <LG tag="h2">{pane.name}</LG>
+                    <MD>{pane.content}</MD>
+                  </div>
+                </Pane.Content>
+                {pane.splitters.map(({ providerId, ...splitter }, splitterIndex) => (
+                  <Pane.Splitter
+                    providerId={providerId}
+                    key={`${pane.name}-${splitter.layoutKey}-${splitterIndex}`}
+                    {...splitter}
+                  />
+                ))}
+              </Pane>
+            ))}
+          </div>
+        );
+      }}
+    </PaneProvider>
+  );
+};
+
+export const CardStory: Story<{ columns: IColumns[] }> = ({ columns }) => {
+  const themeContext = useContext(ThemeContext);
+  const { ref, width = 1, height = 1 } = useResizeObserver<HTMLDivElement>();
+
+  // remove 20px from the totalPanesWidth to account for gap size x 2
+  // otherwise there will be a difference between cursor and splitter position
+  return (
+    <PaneProvider
+      id="column-layout"
+      totalPanesWidth={width - 20}
```

Comment:
[nit] it might be nice to put `gap` into a `const gap = 10;` variable and then use `{width - (gap * 2)}` here (and `gap: ${gap}px` below) in order to drive the meaning of `totalPanesWidth`. In other words, express in code rather than relying (only) on the comment above.

Also, while we're at it, it is always best if you can lean into the system with Garden theming. So prefer a `12px`(?) base-4 gap size based on `theme.space.base`.

Same with `Row` above.

Reply:
Fixed in https://github.com/zendeskgarden/react-components/pull/1356/commits/7cfdaf949e0ff030eb133e4ac672aa2403c64b0b